extensions/objects/templates/svg-editor.component.ts

```diff
@@ -1593,8 +1605,10 @@ export class SvgEditorComponent implements OnInit {
   }
 
   setCanvasDimensions(): void {
-    this.canvas.setHeight(this.CANVAS_HEIGHT);
-    this.canvas.setWidth(this.CANVAS_WIDTH);
+    let dimensions = this.value && (
+      this.imagePreloaderService.getDimensionsOfImage(this.value));
+    this.canvas.setHeight(dimensions?.height || this.CANVAS_HEIGHT);
+    this.canvas.setWidth(dimensions?.width || this.CANVAS_WIDTH);
```

Comment:
I'm a bit confused by this code, but I think it can be addressed in subsequent PR. 

I think that we should have an explanatory comment here. Also I'm unsure what `let dimensions = this.value && this.imagePreloaderService.getDimensionsOfImage(this.value);` does?